### ISSUE TITLE
FakeYou API Token

### ISSUE THREAD

**Issue by thenewgessiko** [MysteryPancake/Discord-TTS#1]
Hi, how do i get the fakeyou api token?

**Comment by MarrcosabTTS** [comment]
Is it really needed for this to work?

**Comment by Dealman** [comment]
> Is it really needed for this to work?

No, it's not necessary. But without one, your commands will be put in a queue. There may be other limitations as well.

**Comment by RMBLegend459** [comment]
Hi. I'm lost as to how to get one voice set up in discord. Could you maybe specify the steps. Also the Client step doesn't work or I don't know if I'm doing it wrong. Is it Application ID?

**Comment by RMBLegend459** [comment]
> Hi. I'm lost as to how to get one voice set up in discord. Could you maybe specify the steps. Also the Client step doesn't work or I don't know if I'm doing it wrong. Is it Application ID?

And how do I find my bot apps on the Discord app? Thank you.
